docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +126,360 @@ microservice reports itself as ready.
 |Treats the instance as still starting up; does not check liveness or readiness until the startup probe reports success or times out according to its configuration.
 |===
 
-=== Built-in and Custom Health Checks
 
-==== Built-in Health Checks
-Helidon provides built-in, default checks for each endpoint.
-The built-in liveness checks include various environmental values, such as whether the JVM has detected deadlocks
-or whether there is sufficient heap space. The built-in readiness and startup checks always report `UP`.
+== Configuration
+
```

Comment:
As it is not it's not clear to me how configuration is used. I would prefer to see a small configuration sample where I can replace some properties to configure something else.

Reply:
This part is generated. It is being reworked now.